CONTRIBUTORS.txt

```diff
@@ -272,7 +272,7 @@ contributors:
 - Justin Li <justinnhli@gmail.com>
 - John Kirkham <jakirkham@gmail.com>
 - Jens H. Nielsen <Jens.Nielsen@microsoft.com>
-- James Addison <55152140+jayaddison@users.noreply.github.com>
+- James Addison <jay@jp-hosting.net>
```

Comment:
You need to change [this file](https://github.com/PyCQA/pylint/blob/main/script/.contributors_aliases.json) instead (the content is autogenerated)

Reply:
D'oh; thanks @Pierre-Sassoulas!